ContentProtection/BasicPlayReady/Program.cs

```diff
@@ -558,9 +555,28 @@ private static ContentKeyPolicyPlayReadyConfiguration ConfigurePlayReadyLicenseT
             string locatorName,
             string contentPolicyName)
         {
-            StreamingLocator locator = await client.StreamingLocators.GetAsync(resourceGroup, accountName, locatorName);
+            bool locatorExists = true;
+            StreamingLocator locator = null;
 
-            if (locator != null)
+            // Let's check if the locator exists already
+            try
+            {
+                locator = await client.StreamingLocators.GetAsync(resourceGroup, accountName, locatorName);
+            }
+            catch (ErrorResponseException ex)
```

Comment:
nit: This whole block can be simplified to 

catch (ErrorResponseException ex) when (ex.Response.StatusCode == NotFound)
{
    locatorExists = false
}